ui/selectors/metamask-notifications/authentication.ts

```diff
@@ -1,8 +1,8 @@
 import { createSelector } from 'reselect';
-import type { AuthenticationControllerState } from '../../../app/scripts/controllers/authentication/authentication-controller';
+import type { AuthenticationController } from '@metamask/profile-sync-controller';
 
 type AppState = {
-  metamask: AuthenticationControllerState & {
+  metamask: AuthenticationController.AuthenticationControllerState & {
```

Comment:
Yep I get it, this syntax is weird. We were fighting to get multiple file exports working for this package, but ran into issues and time constraints.

In near future we will support multiple exports. E.g.

```ts
import { AuthenticationControllerState } from '@metamask/profile-sync-controller/auth'
```